Docs/Documentation/Configuration.md

```diff
@@ -101,6 +101,8 @@ NOTE: SOME keys were hidden in this doc page, please refer to `vendor/cakedc/use
             'ensureActive' => false,
             // default role name used in registration
             'defaultRole' => 'user',
+             // show verbose error to users
+            'ShowVerboseError' => false,
```

Comment:
Config should be in camelCase form, showVerboseError instead of ShowVerboseError